guides/common/modules/proc_migrating-project-or-smartproxy-using-backup.adoc

```diff
@@ -0,0 +1,44 @@
+[id="migrating-{project-context}-or-proxy-using-backup_{context}"]
+= Migrating {Project} or {SmartProxy} to {EL-abbr}{nbsp}9
+
+You can migrate your existing {ProjectServer} and {SmartProxyServer} from {EL}{nbsp}8 to a freshly installed {EL}{nbsp}9 system.
+ifdef::satellite[]
+The migration involves creating a backup of the existing {ProjectServer} and {SmartProxyServer}, which you then restore or clone on the new {EL}{nbsp}9 system.
+[NOTE]
+====
+You can use cloning for {ProjectServer} backups but not for {SmartProxyServer} backups.
+====
+endif::[]
+ifndef::satellite[]
+The migration involves creating a backup of the existing {ProjectServer} and {SmartProxyServer}, which you then restore on the new {EL}{nbsp}9 system.
+endif::[]
+
+.Procedure
+. Perform a full backup of your {ProjectServer} or {SmartProxy}.
+This is the source {EL}{nbsp}8 server that you are migrating.
+For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin#Performing_a_Full_Backup_admin[Performing a full backup of {ProjectServer} or {SmartProxyServer}] in _{AdministeringDocTitle}_.
+. Deploy a system with {EL}{nbsp}9.
+This is the target server.
+Install {ProjectServer} or {SmartProxyServer} on the target {EL}{nbsp}9 system:
+ifeval::[ "{mode}" == "connected" ]
+** To install {ProjectServer}, see {InstallingServerDocURL}[_{InstallingServerDocTitle}_].
+endif::[]
+ifeval::[ "{mode}" == "disconnected" ]
+** To install {ProjectServer}, see {InstallingServerDisconnectedDocURL}[_{InstallingServerDisconnectedDocTitle}_].
+endif::[]
```

Comment:
Sorry that I didn't catch that earlier.

Reply:
I have added a line mentioning this and added some conditionals for builds. @Lennonka could you review the changes?

Reply:
The doc for restore you're linking to already starts with "you need to install Satellite", so I wonder if we even need to mention installation here, or rather say:
1. create a backup
2. create an empty el9 machine
3. restore/clone/whatever

Reply:
Perhaps it would be better to split the migration into two modules: _Migrating ... by using backup_ and _Migrating ... by cloning_. It isn't necessary to do a backup when cloning, is it? @evgeni @AkshayGadhaveRH

Reply:
a backup is the input to clone, so no, it is required.

essentially, the steps you do on the SOURCE are identical for backup and clone, the steps you do on the TARGET are different.